modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc

```diff
@@ -7,7 +7,40 @@
 
 *Prerequisites*
 
-* You have first installed the  link:{kn_cli_install_url}[Knative CLI].
+* You have first installed the link:{kn_cli_install_url}[Knative CLI].
+* link:{java_install_url}[Java] {java_min_version} is installed.
+* link:{maven_install_url}[Maven] {maven_min_version} or later is installed.
```

Comment:
Are you sure maven and Java are required? The run command will execute everything within the container that has these requirements already.

Reply:
Wanted to keep it in sync with upstream, but yea will remove it and sent upstream pr too.

Reply:
add. I think the idea in upstream is that there are quarkus commands being shown in the guide

Reply:
Sure, that's another thing we are going to remove.